decorum/decorum.py

```diff
@@ -63,6 +64,10 @@ def init(self, *args, **kwargs):
 
 
 def decorator(cls):
+    warnings.warn("decorum.decorator() will be deprecated. "
```

Comment:
Ok. I updated the pull-request:

* restored decorum.decorator in README, in a `.. note::` block
* removed the deprecation warning